docs/UserGuide.md

```diff
@@ -211,12 +226,11 @@ Found the following study spots matching “library”:
 
 Deletes the specified study spot from the StudyTracker.
 
-Format: `delete n/NAME*`
+**Format:** `delete n/NAME*`
 
 * Deletes the study spot with the specified **name**.
 
-
-Examples:
+**Examples:**
 ```
 delete n/COM1
 COM1 has been deleted.
```

Comment:
actual output: `Deleted study spot: Central Library; Rating: 3; Operating Hours: Weekdays: 0900-2200, Weekends: 0900-2200; Address: Next to FASS; Favourite: false; Tags: [cold][quiet]; Amenities: [wifi][charger][aircon]`

^ just a random studyspot i deleted